lib/hunter/commands/hunt.rb

```diff
@@ -211,7 +211,7 @@ def process_packet(data:)
 
         # Have to do this last because it depends on the parsed list being up
         # to date
-        apply_to_node(node) if @added
+        apply_to_node(node) if @options.auto_apply && @added
```

Comment:
Sorry, I missed something in my bugfix; I think this should include a check for `dest`. If we haven't added the new node to the parsed list, it won't exist for Flight Profile to apply to.

```ruby
if dest == parsed && @options.auto_apply && @added
  apply_to_node(node)
end
```

Reply:
I didn't think of that, good point